Plutarch/Internal.hs

```diff
@@ -344,6 +346,14 @@ instance ToJSON Config where
           <> ("logLevel" .= ll)
           <> ("tracingMode" .= tm)
 
+-- | @since 1.6.0
```

Comment:
Not sure if this `@since` is right?

Reply:
Does the changelog disagree? If so, it should match the changelog.